var/spack/repos/builtin/packages/py-pyside2/package.py

```diff
@@ -69,6 +69,21 @@ class PyPyside2(PythonPackage):
     depends_on("libxslt@1.1.19:", when="+doc", type="build")
     depends_on("py-sphinx", when="+doc", type="build")
 
+    def patch(self):
+        filter_file(
+            r"\=\${shiboken_include_dirs}",
```

Comment:
If you add `string=True` you won't need to escape special chars

Reply:
I have updated to used `string=True`